openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -121,17 +121,6 @@
   border-radius: 6px;
 }
 
-// Text alignment
```

Comment:
Why did we remove these stylings?

Reply:
It was duplicate, that's why we have removed it.